docs/how-to-backup-your-Hubs-instance.md

```diff
@@ -1,132 +1,70 @@
 ---
 id: how-to-backup-your-Hubs-instance
 title: How to back up your Hubs instance
-description: Information on how to backup and restore your Kubernetes node on DigitalOcean, and thereby your Hubs instance.
+description: Information on how to backup and restore your Hubs instance.
 ---
 
-These instructions are written for users, newbies, or non-developers. These instructions only apply to single-node clusters, where data is stored on the node ("hostPath" volumes).  If you arrived here from the [Beginner’s Guide](./beginners-guide-to-CE.md) which guided you generally to a one node setup, you are fine to follow these.
+These instructions are written for users, newbies, or non-developers.
 
 ### Why would I need to create backups of my Hubs instance?
 
-The Beginner’s Guide instructions and Hubs code have persistent volumes already turned on to store all of your data between DigitalOcean (DO) pod reboots. But if your node(s) get turned off or if DO screws up, you might lose all of your data.  It has been known to happen.
+Backups allow you to have copies so that you can restore your Hubs instance in case anything goes wrong. This will save all of your instance, things like your avatars, and your projects in Spoke, your assets in Spoke, and will save pinned objects in room in the location where you pinned them.
```

Comment:
Looking at this again now, maybe taking out the "will save" before "pinned objects" flows a bit better?
```suggestion
Backups allow you to have copies so that you can restore your Hubs instance in case anything goes wrong. This will save all of your instance, things like your avatars, and your projects in Spoke, your assets in Spoke, and pinned objects in room in the location where you pinned them.
```